applications/StructuralMechanicsApplication/custom_response_functions/structural_response_function_factory.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import print_function, absolute_import, division
```

Comment:
Is it fine that the folder "custom_response_functions" contains both c++ and python files? Or should we put them in the folder with the python scripts to be consistent?

Reply:
Here i don have a strong opinion. In the last discussions we found that it would be a clean solution to put everything related to response functions into this folder.

Maybe we can make an exception for the python files, because that would also make this "hack" unnecessary:

https://github.com/KratosMultiphysics/Kratos/blob/90960015698874f6273d697479ef7f706389f962/applications/StructuralMechanicsApplication/StructuralMechanicsApplication.py#L19

@philbucher what do you say?

Reply:
2 files in this PR, one additional for the adjoints (adjoint_solver)

Reply:
I also don't have a strong opinion, but a tendency for the non-hacky version.

Reply:
then I dont see a problem with putting it with the other scripts